docs/khaas/advanced-configuration.md

```diff
@@ -0,0 +1,51 @@
+# Advanced configuration
+
+This section covering all the flags available to use KubeHound. If you don't want to specify the bucket every time, you can set it up in your local config file (`./kubehound.yaml` or `$HOME/.config/kubehound.yaml`). See [getting started with KHaaS](./getting-started.md)
```

Comment:
```suggestion
This section covers all the available flags in KubeHound. If you don't want to specify the bucket every time, you can set it up in your local config file (`./kubehound.yaml` or `$HOME/.config/kubehound.yaml`). See [getting started with KHaaS](./getting-started.md)
```

Reply:
I think the part with the bucket conf file does not really belong here. Reading it here I was expecting the rest of the document to "teach" me how to get rid of all the flags, and then the subparts explain how to deploy the collector/ingestor.

Maybe in a `config files` sub-section?